desktop/cmp/record/RecordActionBar.js

```diff
@@ -45,7 +45,7 @@ export class RecordActionBar extends Component {
         super(props);
         this.actions = props.actions
             .filter(Boolean)
-            .map(it => new RecordAction(it));
+            .map(it => it instanceof RecordAction ? it : new RecordAction(it));
```

Comment:
Same danger here as in StoreContextMenu of the possibility of multiple UI components sharing the same backing RecordAction.